TST Google options test_expiry_dates test is broken

```
======================================================================
FAIL: test_expiry_dates (pandas_datareader.tests.test_google_options.TestGoogleOptions)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/femtotrader/pandas-datareader/pandas_datareader/tests/test_google_options.py", line 54, in test_expiry_dates
    self.assertTrue(len(dates) > 10)
AssertionError: 
-------------------- >> begin captured logging << --------------------
requests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): www.google.com
requests.packages.urllib3.connectionpool: DEBUG: "GET /finance/option_chain?q=GOOG&expd=3&expm=1&expy=2016&output=json HTTP/1.1" 200 None
--------------------- >> end captured logging << ---------------------
```

--- a/pandas_datareader/tests/test_google_options.py
+++ b/pandas_datareader/tests/test_google_options.py
@@ -51,7 +51,7 @@ class TestGoogleOptions(tm.TestCase):
             dates = self.goog.expiry_dates
         except RemoteDataError as e:  # pragma: no cover
             raise nose.SkipTest(e)
-        self.assertTrue(len(dates) > 10)
+        self.assertTrue(len(dates) > 6)
 
     def test_get_call_data(self):
         with tm.assertRaises(NotImplementedError):

